healthScore/requirements.txt

```diff
@@ -11,6 +11,8 @@ Django==4.2
 filelock==3.13.1
 idna==3.6
 jmespath==1.0.1
+mysql-connector==2.2.9
+mysql-connector-python==8.3.0
```

Comment:
Please remove these two. We don't need these - it causes some deployment issues when run `eb deploy`